TrainingApp/Controller/HomeViewController.swift

```diff
@@ -50,6 +86,35 @@ class HomeViewController: UIViewController {
         }
     }
     
+    private func getData(from url: String) {
+            let task = URLSession.shared.dataTask(with: URL(string: url)!, completionHandler: { data, response, error in
+                
+                guard let data = data, error == nil else {
+                    print("Something went wrong")
+                    return
+                }
+                
+                var result: [Field]?
+                do {
+                    result = try JSONDecoder().decode([Field].self, from: data)
+                    print(result!)
```

Comment:
remove print()

Reply:
Don't use !
It is potential crash

Read smth about ways to unwrap an optional in swift

Reply:
Done 👍
Alright, I'll look online for another solution to unwrap an optional.